packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -246,8 +245,7 @@ export default function ShadowsEditPanel() {
 							</FlexItem>
 							<FlexItem>
 								<Button
-									// TODO: Switch to `true` (40px size) if possible
-									__next40pxDefaultSize={ false }
+									__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> shadows --> add shadow and rename it, a modal would be opened and check for `save` button.

| Before | After     |
| :-------- | :------- |
| <img width="625" alt="shadow-rename-save-before" src="https://github.com/user-attachments/assets/26ffcf65-ea53-44ff-b3e9-67b02fa69ab7"> | <img width="625" alt="shadow-rename-save-after" src="https://github.com/user-attachments/assets/2f10817d-02e9-4ef8-b886-2a00f2224c58"> |